FAQ.md

```diff
@@ -67,3 +67,12 @@ This is an issue with rpm-ostree image-based systems generally, and not specific
 The functionality that provides this, called GHNS, is disabled by default due to the risk posed by the installation of potentially damaging or malicious scripts. This has caused [real damage](https://blog.davidedmundson.co.uk/blog/kde-store-content/). 
 
 If you still want to enable this functionality, run `ujust toggle-ghns`
+
+#### Cannot connect to the internet after boot
```

Comment:
this should be in the form of a question. also it should be more specific, like "my system time is way off, why won't it auto update?"